src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier randomColor = new ColorSupplier();
+        Random randomNum = new Random();
+
+        randomBall.number = randomNum.nextInt(101);

```

Comment:
Direct access to the `number` field of `Ball` class is not recommended. It's better to use setter methods to encapsulate the logic of setting the field values. If the `Ball` class doesn't have a setter method for the `number` field, consider adding it.